pkg/datastore/insightmilestonestore_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datastore
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/pipe-cd/pipe/pkg/insight"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
```

Comment:
haha, the order is opposite, and curious about why it wasn't detected by golinter plugin.